src/index.html

```diff
@@ -7,12 +7,55 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__image"
+        src="./images/imac.jpeg"
+        alt="'iMac with Retina 5K display"
```

Comment:
The 'alt' attribute value should not contain unnecessary quotes at the beginning and the end. It should be `alt="iMac with Retina 5K display"`.